website/landing/components/common/Install.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, Text } from ".";
+
+const DEFAULT_COMMAND = "npm install @codesandbox/sandpack-react";
+
+interface InstallProps {
+  command?: string;
+}
+export const Install: React.FC<InstallProps> = ({
+  command = DEFAULT_COMMAND,
+}) => {
+  // TODO: feedback on copy (?)
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(command);
+  };
+
+  return (
+    <Box
+      css={{
+        alignItems: "center",
+        display: "flex",
+        gap: "12px",
+      }}
+    >
+      <Text
+        css={{
+          fontWeight: "$normal",
+          fontSize: "16px",
+          lineHeight: "19px",
+          textAlign: "center",
+          letterSpacing: "-0.05em",
+
+          "@bp1": {
+            fontSize: "18px",
+            lineHeight: "22px",
+          },
+
+          "@bp2": {
+            fontSize: "24px",
+            lineHeight: "29px",
+          }
+        }}
+      >
+        {command}
+      </Text>
+      <Button
+        aria-label="Copy to cliboard"
+        css={{
+          color: "$darkTextPrimary",
+          flexShrink: "0",
+          height: "12px",
+          width: "12px",
+        }}
+        onClick={copyToClipboard}
+      >
+        <svg fill="none" height="100%" viewBox="0 0 12 13" width="100%">
```

Comment:
I'd split it into a different file too, in order to make the cache assets more effective